src/docfx/cli/Program.cs

```diff
@@ -103,36 +107,47 @@ private static (string command, string docset, CommandLineOptions options) Parse
+                    syntax.DefineOption("v|verbose", ref options.Verbose, "Enable diagnostics console output.");
+                    syntax.DefineParameter("docset", ref docset, "Docset directory that contains docfx.yml/docfx.json.");
 
-                // Watch command
-                syntax.DefineCommand("watch", ref command, "Previews a docset and watch changes interactively.");
-                syntax.DefineOption("locale", ref options.Locale, "The locale of the docset to build.");
-                syntax.DefineOption("port", ref options.Port, "The port of the launched website.");
-                syntax.DefineOption("no-restore", ref options.NoRestore, "Do not restore the docset before building.");
-                syntax.DefineOption("v|verbose", ref options.Verbose, "Enable diagnostics console output.");
-                syntax.DefineParameter("docset", ref docset, "Docset directory that contains docfx.yml/docfx.json.");
-            });
+                    // Build command
+                    syntax.DefineCommand("build", ref command, "Builds a docset.");
+                    syntax.DefineOption("o|output", ref options.Output, "Output directory in which to place built artifacts.");
+                    syntax.DefineOption("legacy", ref options.Legacy, "Enable legacy output for backward compatibility.");
+                    syntax.DefineOption("locale", ref options.Locale, "The locale of the docset to build.");
+                    syntax.DefineOption("no-restore", ref options.NoRestore, "Do not restore the docset before building.");
+                    syntax.DefineOption("v|verbose", ref options.Verbose, "Enable diagnostics console output.");
+                    syntax.DefineParameter("docset", ref docset, "Docset directory that contains docfx.yml/docfx.json.");
 
-            options.Locale = options.Locale?.ToLowerInvariant();
-            return (command, docset, options);
+                    // Watch command
+                    syntax.DefineCommand("watch", ref command, "Previews a docset and watch changes interactively.");
+                    syntax.DefineOption("locale", ref options.Locale, "The locale of the docset to build.");
+                    syntax.DefineOption("port", ref options.Port, "The port of the launched website.");
+                    syntax.DefineOption("no-restore", ref options.NoRestore, "Do not restore the docset before building.");
+                    syntax.DefineOption("v|verbose", ref options.Verbose, "Enable diagnostics console output.");
+                    syntax.DefineParameter("docset", ref docset, "Docset directory that contains docfx.yml/docfx.json.");
+                });
+
+                options.Locale = options.Locale?.ToLowerInvariant();
+                return (command, docset, options);
+            }
+            catch (ArgumentSyntaxException ex)
+            {
+                Log.Write(ex);
```

Comment:
this changed the existing command line experience

Reply:
![image](https://user-images.githubusercontent.com/19990166/57211208-0a894000-7012-11e9-89e2-6033cce33da8.png)

![image](https://user-images.githubusercontent.com/19990166/57211203-00ffd800-7012-11e9-897e-a54b6c6419e0.png)

![image](https://user-images.githubusercontent.com/19990166/57211225-1aa11f80-7012-11e9-8102-2a60254b149a.png)